pkg/kubectl/cmd/apiresources/apiresources.go

```diff
@@ -101,10 +101,13 @@ func NewCmdApiResources(f cmdutil.Factory, ioStreams genericclioptions.IOStreams
 	return cmd
 }
 
-func (o *ApiResourcesOptions) Validate(cmd *cobra.Command) error {
+func (o *ApiResourcesOptions) Complete(cmd *cobra.Command, args []string) error {
```

Comment:
That's not acceptable. You can't change name of this function, you need to add new `Complete` method and follow the patterns from other methods.